src/main/java/com/minecolonies/coremod/colony/jobs/JobEnchanter.java

```diff
@@ -0,0 +1,135 @@
+package com.minecolonies.coremod.colony.jobs;
+
+import com.minecolonies.api.client.render.modeltype.BipedModelType;
+import com.minecolonies.api.colony.ICitizenData;
+import com.minecolonies.api.colony.jobs.ModJobs;
+import com.minecolonies.api.colony.jobs.registry.JobEntry;
+import com.minecolonies.api.util.BlockPosUtil;
+import com.minecolonies.coremod.entity.ai.basic.AbstractAISkeleton;
+import com.minecolonies.coremod.entity.ai.citizen.enchanter.EntityAIWorkEnchanter;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+public class JobEnchanter extends AbstractJob
+{
+    /**
+     * The tag to store the building he is currently draining from.
+     */
+    private static final String TAG_BUILDING_TO_DRAIN = "buildingtodrain";
```

Comment:
Constants and capitalisation

Reply:
Same thing, I want to go into the direction of all lower case of vanilla, since that's stongly enforced in 1.14